### ISSUE TITLE
npm run generate error on routes

### ISSUE THREAD

**Issue by nusson** [nuxt-community/router-module#2]
Since I used your module, I'm not able to generate a static website `yarn run generate'

here the error
```
TypeError: Cannot read property 'forEach' of undefined
    at flatRoutes (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/dist/nuxt.js:1288:9)
    at Generator._callee2$ (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/lib/builder/generator.js:62:66)
    at tryCatch (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:299:22)
    at Generator.prototype.(anonymous function) [as next] (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (/Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/dist/nuxt.js:954:30)
    at /Users/nicolashusson/www/shingrix/htdocs/node_modules/nuxt/dist/nuxt.js:965:13
```

indeed, in node_modules/nuxt/dist/nuxt.js, function flatRoutes(router)
router is undefined...

**Comment by robyedlin** [comment]
+1

**Comment by Atinux** [comment]
It's fixed with [v1.0.1](https://github.com/nuxt-community/router-module/releases/tag/v1.0.1) :confetti_ball:
